test/nonregression/test_run_pipelines.py

```diff
@@ -181,53 +188,11 @@ def test_run_T1VolumeDartel2MNI(cmdopt):
     clean_folder(join(root, 'out', 'caps'), recreate=False)
 
 
-def test_run_T1VolumeNewTemplate(cmdopt):
```

Comment:
Is this test removed ? Why ?

Reply:
The `T1Volume[NewTemplate]` class has disappeared so its associated tests are not needed anymore :)

Reply:
But we still need to test that t1-volume is able to perform !